terraform/deployments/cluster-services/main.tf

```diff
@@ -56,6 +56,7 @@ provider "helm" {
 }
 
 locals {
+  apps_ns                = "apps"
```

Comment:
It doesn't smell right to me that the cluster Terraform stuff should be aware of which namespace the apps are running in - I'd say that's a fairly major layering violation between base cluster services and userland. I'd suggest making this a var and renaming it to something like `argo_workflow_namespaces` to reflect what it's actually doing (and make it a list and merge it on argo.tf line 99. It'd be very unfortunate if someone were to use `apps_ns` for other things in this TF module.